pants.toml

```diff
@@ -115,6 +115,7 @@ pants_plugins_interpreter_constraints = "CPython==3.9.*"
 
 # For tools, we have to include python versions for BOTH st2 and pants-plugins
 tool_interpreter_constraints = "CPython>=3.8,<3.10"
+flake8_interpreter_constraints = "CPython>=3.8.1,<3.10"
```

Comment:
flake8 requires python `>=3.8.1`. It would probably be better to apply that python version constraint to all our lockfiles, but this PR already touches several lockfiles and I didn't want to regenerate all of them.